src/components/ChatApp/MessageSection/DialogSection.js

```diff
@@ -111,9 +126,9 @@ const DialogSection = props => {
             </LinkedinShareButton>
           </div>
         </div>
-        <Close style={closingStyle} onClick={toggleShareClose} />
+        <Close style={closingStyle} onClick={actions.closeModal} />
       </Dialog>
-      <Dialog fullWidth={true} maxWidth={'sm'} open={tour}>
+      <Dialog fullWidth={true} maxWidth={'sm'} open={!visited}>
```

Comment:
Let's have a switch case here, instead of writing code for rendering each Dialog. And just write 1 Dialog code block on the return block.

Reply:
Do you mean for share, tour dialog only? @akshatnitd

Reply:
@akshatnitd awesome idea 👍